docs/CHANGELOG.md

```diff
@@ -9,6 +9,124 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+
+##### Jobs/tasks
+
+By default, all jobs/tasks will continue to use the default chain (specified by `ETH_CHAIN_ID`). However, the following jobs now allow an additional `evmChainID` key in their TOML:
+
+- VRF
+- DirectRequest
+- Keeper
+- OCR
+- Fluxmonitor
+
+You can pin individual jobs to a particular chain by specifying the `evmChainID` explicitly. Here is an example job to demonstrate:
+
+```toml
+type            = "keeper"
+evmChainID      = 3 
+schemaVersion   = 1
+name            = "example keeper spec"
+contractAddress = "0x9E40733cC9df84636505f4e6Db28DCa0dC5D1bba"
+externalJobID   = "0EEC7E1D-D0D2-476C-A1A8-72DFB6633F49"
+fromAddress     = "0xa8037A20989AFcBC51798de9762b351D63ff462e"
+```
+
+The above keeper job will _always_ run on chain ID 3 (Ropsten) regardless of the `ETH_CHAIN_ID` setting. If no chain matching this ID has been added to the chainlink node, the job will error.
+
+In addition, you can also specify `evmChainID` on certain pipeline tasks. This allows for cross-chain requests, for example:
+
+```toml
+type                = "directrequest"
+schemaVersion       = 1
+evmChainID          = 42
+name                = "example cross chain spec"
+contractAddress     = "0x613a38AC1659769640aaE063C651F48E0250454C"
+externalJobID       = "0EEC7E1D-D0D2-476C-A1A8-72DFB6633F90"
+observationSource   = """
+    decode_log   [type=ethabidecodelog ... ]
+    ...
+    submit [type=ethtx to="0x613a38AC1659769640aaE063C651F48E0250454C" data="$(encode_tx)" minConfirmations="2" evmChainID="3"]
+    decode_log-> ... ->submit;
+"""
+```
+
+In the example above (which excludes irrelevant pipeline steps for brevity) a log can be read from the chain with ID 42 (Kovan) and a transaction emitted on chain with ID 3 (Ropsten).
+
+Tasks that support the `evmChainID` parameter are as follows:
+
+- `ethcall`
+- `estimategaslimit`
+- `ethtx`
+
+###### Defaults
+
+If the job- or task-specific `evmChainID` is _not_ given, the job/task will simply use the default as specified by the `ETH_CHAIN_ID` env variable.
+
+Generally speaking, the default config values for each chain are good enough. But in some cases it is necessary to be able to override the defaults on a per-chain basis.
+
+This used to be done via environment variables e.g. `MINIMUM_CONTRACT_PAYMENT`.
+
+These still work, but if set they will override that value for _all_ chains. This may not always be what you want. Consider a node that runs both Matic and Mainnet. You may want to set a higher value for `MINIMUM_CONTRACT_PAYMENT` on Mainnet, due to the more expensive gas costs. However, setting `MINIMUM_CONTRACT_PAYMENT` using env variables will set that value for _all_ chains including matic.
+
+To help you work around this, Chainlink now supports setting per-chain configuration options.
+
+(the full list of chain-specific configuration options can be found in `core/chains/evm/config.go`)
+
+TODO: https://app.clubhouse.io/chainlinklabs/story/15455/add-cli-api-for-setting-chain-specific-config
```

Comment:
To remove from changelog?

Reply:
I will update the changelog once this card is done